### PR TITLE
Unlcaim Prrrs Over an Hour Old [resolves #2]

### LINKED ISSUE #2: If PR review is not completed within an hour of being claimed the PR moves back to "waiting for review"

### PR DESCRIPTION
**Pull request by Anasauce**
- wrote function to unclaim all PRs that have not been completed within an hour and return them to the que
- worker script to run uncliam old PRs function every 10 minutes via Heroku scheduler
- created Procfile for syncing with Heroku Scheduler
- test coverage for this feature had been written

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+web: node server/index.js
+worker: node server/worker.js
+dev: webappatron index.js
```
Comment (deadlyicon) [inline Procfile]: i didn't know you could do `dev` and `profile`. What do these do?

File: bin/import-database-from-production [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```
Comment (deadlyicon) [inline bin/import-database-from-production]: why is this in your PR? weird

File: browser/components/molecules/PendingPrrrs/index.js [hunk 3]
```diff
@@ -53,8 +53,8 @@ export default class PendingPrrrs extends Component {
     const prrrs = this.props.prrrs
       .filter(prrr => !prrr.claimed_by)
       .sort((a, b) =>
-        moment(b.claimed_at).valueOf() -
-        moment(a.claimed_at).valueOf()
+        moment(a.created_at).valueOf() -
```
Comment (deadlyicon) [inline browser/components/molecules/PendingPrrrs/index.js]: also this should not be in your PR. please rebase off master

File: circle.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+machine:
```
Comment (deadlyicon) [inline circle.yml]: also this. so weird. Please rebase this branch

File: server/commands/index.js [hunk 5]
```diff
@@ -202,6 +202,21 @@ export default class Commands {
...
+      .update({
+        claimed_by: null,
+        claimed_at: null,
+        updated_at: new Date,
+      })
+      .whereRaw(`claimed_at <= NOW() - '1 hour'::INTERVAL`)
+      .whereNotNull('claimed_by')
+      .whereNotNull('claimed_at')
+      .whereNull('completed_at')
+      .returning('*')
```
Comment (deadlyicon) [inline server/commands/index.js]: I dont think we need to return anything from this query

File: Procfile (hunk 1 above)
Comment (Anasauce) [inline Procfile]: uhhh actually that is wrong...oops, taking it out. I actually don't know how to configure the profile.

File: package.json [hunk 6]
```diff
@@ -7,6 +7,7 @@
     "test": "webappatron test",
     "start": "webappatron start",
     "build": "webappatron build",
+    "worker": "node ./build/server/worker.js",
```
Comment (deadlyicon) [inline package.json]: whoppps remove this :P my bad.

**Pull request merged**